City-of-Helsinki/drupal-infofinland
# Add footer about menu and change main menu lang

**What has changed**

- Add footer about menu
- Change main menu lang
- Change nextjs module settings
- Add config ignore for nextjs related things
- Add language selection to admin pages
- Remove municipality block from pages

**How to test**

Open your consumer and nextjs settings in new tabs or write down the values in other way before testing the PR. The values should not change after you have deployed the changes.
consumer: https://drupal-infofinland.docker.so/en/admin/config/services/consumer
nextjs: https://drupal-infofinland.docker.so/en/admin/config/services/next
https://drupal-infofinland.docker.so/en/admin/config/services/next/entity-types


```
Make build
Make shell
drush cim -y
drush cr
```
Go to: `/en/admin/structure/menu` check that both menus exist and main menu lang is finnish.

Check have your consumer and nextjs settings changed. They shouldn't change.
consumer: https://drupal-infofinland.docker.so/en/admin/config/services/consumer
nextjs: https://drupal-infofinland.docker.so/en/admin/config/services/next
https://drupal-infofinland.docker.so/en/admin/config/services/next/entity-types

Check the is language selection in admin pages. You should find it from upper right corner.

Check that municipality block is not on any page e.g. https://drupal-infofinland.docker.so/en/user/1
